test/unit/DrawNoReplTest.cpp

```diff
@@ -0,0 +1,59 @@
+#include <catch.hpp>
```

Comment:
Do you mind combining this and the other test file into a single file named after the class you're testing i.e. two test cases in `DataSetGeneratorTest.cpp`